app/helpers/textual_mixins/textual_refresh_status.rb

```diff
@@ -1,20 +1,29 @@
 module TextualMixins::TextualRefreshStatus
   def textual_refresh_status
-    last_refresh_status = @record.last_refresh_status.titleize
-    if @record.last_refresh_date
-      last_refresh_date = time_ago_in_words(@record.last_refresh_date.in_time_zone(Time.zone)).titleize
+    refresh_status(@record)
+  end
+
+  def textual_refresh_date
+    refresh_date(@record)
+  end
+
+  def refresh_status(record)
+    last_refresh_status = record.last_refresh_status ? record.last_refresh_status.titleize : nil
```

Comment:
Simplified, this can be

```suggestion
    last_refresh_status = record.last_refresh_status&.titleize
```